src/OrchardCore/OrchardCore.Abstractions/Json/Dynamic/JsonDynamicObject.cs

```diff
@@ -39,6 +39,8 @@ public void Merge(JsonNode? content, JsonMergeSettings? settings = null)
         set => SetValue(key, value);
     }
 
+    public T? ToObject<T>() => _jsonObject.ToObject<T>();
```

Comment:
Can we create a base class and move this to it?

Reply:
Probably, but it doesn't buy us much for this single method.

Reply:
Done.